app/views/home/how_government_works.html.erb

```diff
@@ -25,9 +47,9 @@
             <h3>The Prime Minister</h3>
             <p>The <a href="/government/ministers/prime-minister">Prime Minister</a> is head of the UK government. He is ultimately responsible for all policy and decisions. He:</p>
             <ul>
-              <li>oversees the operation of the Civil Service and government agencies</li>
-              <li>appoints members of the government</li>
-              <li>is the principal government figure in the House of Commons</li>
+              <li>oversee the operation of the Civil Service and government agencies</li>
```

Comment:
Yep, i messed that up when re-ordering @bmwelby's commits, will fix with a rebase.